### ISSUE TITLE
/petrespawn auto not work

### ISSUE THREAD

**Issue by andris155** [xXKeyleXx/MyPet#935]
**Bukkit/Spigot/Bungee Build/Version:**
This server is running Paper version git-Paper-962 (MC: 1.11) (Implementing API version 1.11-R0.1-SNAPSHOT)

**MyPet Build/Version:**
MyPet-Premium 2.3.2

**Do you use the premium version? (this wont change anything for you)**
yes

**What is the expected result?**
Pet respawn auto

**What happens instead of that?**
Pet not respawn auto

**Error:**
none

**Comment by xXKeyleXx** [comment]
This works fine for me. This requires an economy plugin.
Maybe your selected respawn time is bigger than the actual time the pet takes to respawn

**Comment by asdlol911** [comment]
@xXKeyleXx the problem is that, we use the right command (/petrespawn auto or /pr auto both), the chat shows that, we activated it, but it won1t revive the pet instantly, we still have to wait till the pet respawns.
Just like my pet, (Villager) died after I activated the auto respawn, and the chat said, I have to wait 300 sec till my pet respawns.
So,,,the petrespawn command works only on chat, till it shows itself, but unable to activate in game.
